File: api-java/FocusPlanner/src/main/java/com/example/tfg/enums/TaskStatus.java
package com.example.tfg.enums;

import lombok.Getter;

@Getter
public enum TaskStatus {
    PENDING("PENDING"),
    COMPLETED("COMPLETED"),
    EXPIRED("EXPIRED"),
    COMPLETED_OR_EXPIRED("COMPLETED_OR_EXPIRED");

    private final String value;

    TaskStatus(String value) {
        this.value = value;
    }

}

File: app-kotlin/app/src/main/java/com/example/focus_planner/data/model/task/TaskStatus.kt
package com.example.focus_planner.data.model.task

enum class TaskStatus(val value: String, val displayName: String) {
    PENDING("PENDING", "Pendiente"),
    COMPLETED("COMPLETED", "Completada"),
    EXPIRED("EXPIRED", "Expirada"),
    COMPLETED_OR_EXPIRED("COMPLETED_OR_EXPIRED", "Completada y Expirada")
}

enum class TaskPriority(val value: String, val displayName: String) {
    LOW("LOW", "Baja"),
    MEDIUM("MEDIUM", "Media"),
    HIGH("HIGH", "Alta")
}
